bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeper.java

```diff
@@ -462,13 +463,13 @@ public void asyncCreateLedger(final int ensSize,
      *          createCallback implementation
      * @param ctx
      *          optional control object
+     * @param customData
+     *          optional customData that holds user specified metadata
      */
 
-    public void asyncCreateLedger(final int ensSize,
-                                  final int writeQuorumSize,
-                                  final int ackQuorumSize,
-                                  final DigestType digestType,
-                                  final byte[] passwd, final CreateCallback cb, final Object ctx) {
+    public void asyncCreateLedger(final int ensSize, final int writeQuorumSize, final int ackQuorumSize,
```

Comment:
This will break binary backward compatibility. You should keep original method and add a overload method.